google-cloud-spanner/src/main/java/com/google/cloud/spanner/admin/database/v1/DatabaseAdminSettings.java

```diff
@@ -224,14 +224,13 @@ protected Builder(DatabaseAdminStubSettings.Builder stubSettings) {
       return ((DatabaseAdminStubSettings.Builder) getStubSettings());
     }
 
-    // NEXT_MAJOR_VER: remove 'throws Exception'
     /**
      * Applies the given settings updater function to all of the unary API methods in this service.
      *
      * <p>Note: This method does not support applying settings to streaming methods.
      */
     public Builder applyToAllUnaryMethods(
-        ApiFunction<UnaryCallSettings.Builder<?, ?>, Void> settingsUpdater) throws Exception {
+        ApiFunction<UnaryCallSettings.Builder<?, ?>, Void> settingsUpdater) {
```

Comment:
These files are auto-generated. Are we sure these won't get reverted the next time?

Reply:
Shall I revert this and make the PR handles channel pool only? We can leave the `throws exceptions` to another PR, either auto-generated or manually-written.

Reply:
That SGTM.

Reply:
This change to generated code still needs to be reverted

Reply:
Done.